Rubberduck.CodeAnalysis/QuickFixes/EncapsulateFieldQuickFix.cs

```diff
@@ -3,38 +3,36 @@
 using Rubberduck.Parsing.Inspections.Abstract;
 using Rubberduck.Parsing.Rewriter;
 using Rubberduck.Parsing.VBA;
+using Rubberduck.Refactorings;
 using Rubberduck.Refactorings.EncapsulateField;
 using Rubberduck.SmartIndenter;
-using Rubberduck.UI.Refactorings.EncapsulateField;
 using Rubberduck.VBEditor.SafeComWrappers.Abstract;
 
 namespace Rubberduck.Inspections.QuickFixes
 {
     public sealed class EncapsulateFieldQuickFix : QuickFixBase
     {
-        private readonly IVBE _vbe;
         private readonly RubberduckParserState _state;
+        private readonly IVBE _vbe;
         private readonly IRewritingManager _rewritingManager;
         private readonly IIndenter _indenter;
-
-        public EncapsulateFieldQuickFix(IVBE vbe, RubberduckParserState state, IIndenter indenter, IRewritingManager rewritingManager)
+        private readonly IRefactoringPresenterFactory _factory;
```

Comment:
Wouldn't it be cleaner to put this in a base class that inherits from `QuickFixBase`? Something like `public class PresenterQuickFixBase : QuickFixBase`?